src/lego_robot/main.c

```diff
@@ -13,14 +13,14 @@
 void follow_black_line(int, int);
 void drive(int, int, int);
 void drive_to_start();
+void bring_poms_to_Analysis_Center();
 boolean is_frontal_collision();
 boolean drive_distance(int);
+boolean is_over_black_line = false;
```

Comment:
Is this variable actually getting used? Why is it defined globally and not within scope?